src/test-environment-maker.spec.js

```diff
@@ -0,0 +1,65 @@
+var path = require('path');
+var expect = require('chai').expect;
+var simpleMock = require('../lib/simple-mock/index.js');
+var mockRequire = require('mock-require');
+
+describe('Test environment maker', function() {
```

Comment:
This should have been included in a previous pull request, but it was accidentally forgotten.